tutorials/btp-integration-suite-execute-batch-payload/btp-integration-suite-execute-batch-payload.md

```diff
@@ -0,0 +1,377 @@
+
+
+### Create data source and upload driver
+JDBC adapter supports third party and in-house databases on Neo and Cloud Foundry Business Technology environments. Before you establish the connection to these databases, or setup an integration flow, you need to upload drivers (applicable for HANA, ASE, PostgreSQL only) and add Data Sources.
+
+Refer to [Configure JDBC Drivers](https://help.sap.com/docs/CLOUD_INTEGRATION/368c481cd6954bdfa5d0435479fd4eaf/77c7d9550e12494eb600ec82496ef215.html?locale=en-US&version=Cloud) to understand the process of uploading the drivers and the databases.
+<!-- border -->![Uploading Drivers](Deploy-Driver.png)
+
+Refer to [Managing JDBC Data Sources](https://help.sap.com/docs/CLOUD_INTEGRATION/368c481cd6954bdfa5d0435479fd4eaf/4c873fac537248e58767f74e4a74d867.html?locale=en-US&version=Cloud) to understand the process to create a data source.
+<!-- border -->![Deploying Data Sources](DS-Deploy.png)
+
+### Connect with Receiver/Database via JDBC adapter
+Let's create a basic integration flow to understand the basic function of JDBC adapter. In this basic integration scenario, a sender is connected to the Start event via HTTPS Sender adapter and the End event is connected with receiver via JDBC adapter. This JDBC adapter enables the connection of the integration flow with the receiver's database. It allows you to send, receive, update the data of receiver's database.
+
+  <!-- border -->![Initial Integration Flow](Basic-Integration-Flow.png)
+
+You must set the following properties:
+
+-	For HTTPS adapter: Under **Connection tab** > **Address**: Define your end point.
+
+  <!-- border -->![The HTTPS settings](HTTPS-Setting.png)
+
+To understand about other settings, refer to [HTTPS Sender Adapter](https://help.sap.com/docs/CLOUD_INTEGRATION/368c481cd6954bdfa5d0435479fd4eaf/0ae4a78909c4479cbc3cc414250919de.html?locale=en-US&version=Cloud)
+
+-	For the JDBC adapter: Under **Connection tab** > **JDBC Data Source Alias**, enter the name of database source to which the adapter should connect to.
+
+  <!-- border -->![The JDBC settings](JDBC-Settings.png)
+
+### Create Sample Database
+
+To execute this scenario, let's create a table with few columns.
+
+1. Add a content modifier and then add following query to its **Message Body**. To ensure duplicate row entry is restricted in your database, make one of the columns as primary key.
+
+    `create table samplejdbc (emp_id integer primary key, emp_name varchar(255));`
+
+      <!-- border -->![Add Content Modifier and input Query](Input-Query.png)
+
+2. Deploy this integration flow. We will now try to execute this via [Postman]( https://www.getpostman.com/) (You can use any other HTTPS client application too.)
```

Comment:
Don't say "we" when addressing the author. Say "you" instead!

Reply:
done